app/config/DocumentationController.scala

```diff
@@ -22,14 +22,13 @@ import javax.inject.{Inject, Singleton}
 import play.api.http.HttpErrorHandler
 import play.api.libs.json.Json
 import play.api.mvc.{Action, AnyContent, ControllerComponents}
-import uk.gov.hmrc.api.controllers.{DocumentationController => HmrcDocumentationController}
-
+import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
```

Comment:
Can you please add a space below this and also use the appropriate imports for the JsonErrorHandler in the ErrorHandler and for the BackendController in the AuthorisedController as these throw warnings when compiling.

Reply:
Works fine without a space here, the other two valid changes have been pushed

Reply:
Yep I know it works fine without a space but it's to separate the imports with and the other code. Pretty much add a space i.e. put an extra line to separate the last import and the @Singleton like we do with the package and the imports etc.

Reply:
it doesn't change the functionality of the code

Reply:
Thanks. While you're right that this would affect the functionality of the code, however, I am of the opinion that making it clearer and cleaner, and consistent for continued development is good. Just my opinion.

Reply:
It's my opinion that adding a space here adds no value, and makes it no clearer than before. If you would like to add a space then go ahead, but there are other tasks I'm working on with a greater priority than adding a space for aesthetic purposes.

Reply:
Noted. Will add the line myself.